tests/integration/distributed/test_simple_distributed/test_integration.py

```diff
@@ -3,47 +3,48 @@
 import sys
 import time
 
-from tests.integration.helpers import (
+from tests.helpers import (
     start_docker_compose,
     stop_docker_compose,
     send_flow,
     call_api,
     get_results,
 )
 
-directory = Path('tests/integration/distributed/test_simple_distributed/')
-compose_yml = directory / 'docker-compose.yml'
-flow_yml = directory / 'flow.yml'
-pod_dir = directory / 'pods'
+directory = Path("tests/integration/distributed/test_simple_distributed/")
+compose_yml = directory / "docker-compose.yml"
+flow_yml = directory / "flow.yml"
+pod_dir = directory / "pods"
 
 
 def test_flow():
-    if Path.cwd().name != 'jinad':
-        sys.exit('test_simple_distributed.py should only be run from the jinad base directory')
+    if Path.cwd().name != "jinad":
+        sys.exit(
+            "test_simple_distributed.py should only be run from the jinad base directory"
+        )
 
     start_docker_compose(compose_yml)
 
     time.sleep(10)
 
-    flow_id = send_flow(flow_yml, pod_dir)['flow_id']
+    flow_id = send_flow(flow_yml, pod_dir)["flow_id"]
```

Comment:
Why did we switch back to double quotations?

Reply:
I saved your file after changes and my formatter double quotes everything :(